### DISCUSSION TITLE
How to get rid of unicode characters from your git branch?

### DISCUSSION THREAD

**Discussion by AshuT930** [community/community#55717] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Question

### Select Feature Area

- [X] Issues
- [ ] Projects

### Body

I use git bash for git related activities on windows.

I created a branch, did some changes and committed. When I tried to push, it gives me:

`error "refpath does not exist".`

I then checkout to other branch and retried to checkout to my branch but it says

`error: pathspec 'feature/my-branch' did not match any file(s) known to git. `

on running `git branch`, it lists my branch name as-
`<U+0085><U+0085><U+0085><U+0086>feature/my-branch`

I tried renaming this branch but that also didn't work.
`git branch -m  '<U+0085><U+0085><U+0085><U+0086>feature/my-branch' feature/new-branch`

what is the reason behind this and what could be the possible solution?

### Guidelines

- [X] I have read the above statement and can confirm my post is relevant to the GitHub feature areas [Issues](https://docs.github.com/en/issues/tracking-your-work-with-issues/about-issues) and/or [Projects](https://docs.github.com/en/issues/planning-and-tracking-with-projects/learning-about-projects/about-projects).

**Comment by queenofcorgis** [comment]
Thanks for posting in the GitHub Community, @AshuT930 ! 

We're happy you're here. You are more likely to get a useful response if you are posting your question in the applicable category, the Issues category is solely related to conversations around the GitHub products Issues and Projects. I've gone ahead and moved it for you. Good luck!
